scipy/special/_lambertw.h

```diff
@@ -0,0 +1,162 @@
+/* Implementation of the Lambert W function [1]. Based on MPMath
+ *  Implementation [2], and documentation [3].
+ *
+ * Copyright: Yosef Meller, 2009
+ * Authoer email: mellerf@netvision.net.il
+ *
+ * Distributed under the same license as SciPy
+ * Translated into C++ by SciPy developers, 2023.
+ *
+ * References:
+ * [1] On the Lambert W function, Adv. Comp. Math. 5 (1996) 329-359,
+ *     available online: http://www.apmaths.uwo.ca/~djeffrey/Offprints/W-adv-cm.pdf
+ * [2] mpmath source code, Subversion revision 990
+ *     http://code.google.com/p/mpmath/source/browse/trunk/mpmath/functions.py?spec=svn994&r=992
+ * [3] mpmath source code, Subversion revision 994
+ *     http://code.google.com/p/mpmath/source/browse/trunk/mpmath/function_docs.py?spec=svn994&r=994
```

Comment:
Not a blocking comment but these references no longer work

Reply:
Thanks! Good catch

Reply:
Maybe just link to the current commits on github for these files? i looked through the history and it doesn't seem like these implementations have been touched in the past decade or more.

https://github.com/mpmath/mpmath/blob/c5939823669e1bcce151d89261b802fe0d8978b4/mpmath/functions/functions.py

and

https://github.com/mpmath/mpmath/blob/c5939823669e1bcce151d89261b802fe0d8978b4/mpmath/function_docs.py

Reply:
I updated the links, using internet archive where possible.